logging/config/cluster-logging-maintenance-support.adoc

```diff
@@ -0,0 +1,51 @@
+:context: cluster-logging-unsupported
+[id="cluster-logging-maintenance-and-support"]
+= Maintenance and support
+include::modules/common-attributes.adoc[]
+
+
```

Comment:
No TOC required as there are no modules. Are we not supposed to use `{nbsp} +`?

Reply:
Oh, I didn't notice that there were no modules. I would still add the toc line - I don't think it will render anything if there are no modules, but it's good to have as part of the top template in case modules are added at some point.